packages/appChrome/components/SidebarItem.tsx

```diff
@@ -14,6 +14,11 @@ export interface SidebarItemProps {
 class SidebarItemComponent extends React.PureComponent<SidebarItemProps, {}> {
   public render() {
     const { children, isActive, onClick } = this.props;
+    const dataCy = [
```

Comment:
I can't find the data-cy for `SidebarItem`s.

Reply:
@GeorgiSTodorov 	is right. we need to search for things that are unwrapped. e.g. `Clickable` will be unwrapped, so only it's children are actually rendered. thus we lose our `data-cy`-prop.